crates/apub/src/protocol/activities/block/block_user.rs

```diff
@@ -38,7 +38,9 @@ pub struct BlockUser {
   pub(crate) remove_data: Option<bool>,
   /// block reason, written to mod log
   pub(crate) summary: Option<String>,
+  /// deprecated
```

Comment:
Add a TODO line here, or maybe create another issue to keep track of this.

Reply:
Added todo.